docs/Reference/Decorators.md

```diff
@@ -2,22 +2,20 @@
 
 ## Decorators
 
-The decorators API allows customization of the core Fastify objects, such as the
-server instance itself and any request and reply objects used during the HTTP
-request lifecycle. The decorators API can be used to attach any type of property
-to the core objects, e.g. functions, plain objects, or native types.
-
-This API is *synchronous*. Attempting to define a decoration asynchronously
-could result in the Fastify instance booting before the decoration completes its
-initialization. To avoid this issue, and register an asynchronous decoration,
-the `register` API, in combination with `fastify-plugin`, must be used instead.
-To learn more, see the [Plugins](./Plugins.md) documentation.
-
-Decorating core objects with this API allows the underlying JavaScript engine to
-optimize the handling of server, request, and reply objects. This is
-accomplished by defining the shape of all such object instances before they are
-instantiated and used. As an example, the following is not recommended because
-it will change the shape of objects during their lifecycle:
+The decorators API customizes core Fastify objects, such as the server instance
+and any request and reply objects used during the HTTP request lifecycle. It
+can attach any type of property to core objects, e.g., functions, plain
+objects, or native types.
+
+This API is *synchronous*. Defining a decoration asynchronously could result in
+the Fastify instance booting before the decoration completes. To register an
+asynchronous decoration, use the `register` API with `fastify-plugin`. See the
+[Plugins](./Plugins.md) documentation for more details.
+
+Decorating core objects with this API allows the JavaScript engine to optimize
+handling of server, request, and reply objects. This is done by defining the
+shape of these objects before they are instantiated. The following example is
+not recommended as it changes the shape of objects during their lifecycle:
```

Comment:
I know I wrote it, but I think the original reads better.

Reply:
Reverted in https://github.com/fastify/fastify/pull/5966/commits/09e97e89707841872561d063e89e927a17c2042b